src/csharp/Grpc.Core/Internal/PlatformApis.cs

```diff
@@ -49,11 +49,15 @@ internal static class PlatformApis
 
         static PlatformApis()
         {
-#if NETSTANDARD1_5 || NETSTANDARD2_0
+#if NETSTANDARD
             isLinux = RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
             isMacOSX = RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
             isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
-            isNetCore = RuntimeInformation.FrameworkDescription.StartsWith(".NET Core");
+            isNetCore =
+#if NETSTANDARD2_0
```

Comment:
Why is the `Environment.Version.Major >= 5` only used on NETSTANDARD2_0 and not on netstandard1.5?

I understand that on .NET5,  netstandard2.0 will always be used, but I don't see a good reason not to check `Environment.Version.Major` on netstandard1.5 as well - it won't do any harm and it would make the code simpler? But perhaps there's something I'm missing, please let me know if so.

Reply:
Ok, I will answer myself. Environment.Version.Major is not available on netstandard1.5 it seems and I'd get  `'Environment' does not contain a definition for 'Version'`

Reply:
Yes, `Environment.Version` is not available on older target frameworks.